.php-cs-fixer.php

```diff
@@ -16,15 +16,19 @@
     ->setCacheFile(__DIR__.'/var/.php_cs.cache')
     ->setRules([
         '@PSR2' => true,
-        'blank_line_after_opening_tag' => true,
-        'blank_line_before_statement' => true,
+        '@PhpCsFixer' => true,
+        '@PhpCsFixer:risky' => true,
+        'binary_operator_spaces' => [
+            'operators' => [
+                '=>' => 'align_single_space_minimal',
+            ]
+        ],
         'class_attributes_separation' => ['elements' => ['const' => 'one', 'property' => 'one', 'method' => 'one']],
+        'concat_space' => ['spacing' => 'one'],
         'declare_strict_types' => true,
-        'no_unused_imports' => true,
-        'ordered_imports' => true,
-        'psr_autoloading' => true,
```

Comment:
why you removed those 3? Are they covered by other rules you added?

Reply:
correct, they're part of PhpCsFixer ruleset, see https://cs.symfony.com/doc/ruleSets/PhpCsFixer.html